src/browserenginekit.cs

```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
 		[Field ("BEAccessibilityValueChangedNotification")]
 		Changed,
 	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface BEAccessibilityTextMarker : INSCopying, INSSecureCoding
+	{
+	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface BEAccessibilityTextMarkerRange : INSCopying, INSSecureCoding
+	{
+		[Export ("startMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker StartMarker { get; set; }
+
+		[Export ("endMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker EndMarker { get; set; }
+	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[Protocol]
+	interface BEAccessibilityTextMarkerSupport
```

Comment:
Any particular reason not to use new style protocol support?

Reply:
Besides marking this protocol as not requiring backwards code gen, I'm not sure what else you suggest?